src/mocks/routes/handlers.js

```diff
@@ -0,0 +1,9 @@
+import { rest } from "msw";
+import { productController } from "mocks/controllers";
+
+export const handlers = [
+  rest.get("/products/all", productController.getAll),
```

Comment:
export const apiUrl = ({ id, sourceId }: Params = {}) => ({
  prodcuts: {
    getAll: '/api/products',
    edit: `/api/secret-stores/${id}`,
   ...
  },
})


`Products: 

1. List of Products /products (GET)
2. Create Product /products (POST)
3. Delete Product /products (DELETE)
4. Get Product /product/:id (GET)
5. Edit Product /product/:id (PUT/PATCH)
`

Reply:
fixed